src/lvalue.rs

```diff
@@ -238,6 +241,18 @@ impl<'ctx> LValue<'ctx> {
             },
         }
     }
+
+    #[cfg(feature = "master")]
+    pub fn get_name(&self) -> Result<Option<&'ctx str>, Utf8Error> {
+        unsafe {
+            let str = gccjit_sys::gcc_jit_lvalue_get_name(self.ptr);
+            if str.is_null() {
+                Ok(None)
+            } else {
+                Ok(Some(CStr::from_ptr(str).to_str()?))
```

Comment:
Is it possible to get a UTF-8 error here?
Since we get a name which is ascii?

Reply:
Likely not, and I suppose rust would have already translated names at this point with mangling etc.